project-management-application-web/src/templates/typescript-angular/api.service.mustache

```diff
@@ -56,7 +56,7 @@ export class {{classname}} implements {{classname}}Interface {
 export class {{classname}} {
 {{/withInterfaces}}
 
-    protected basePath = '{{{basePath}}}';
+    protected basePath = 'http://localhost:8080';
```

Comment:
hardcoding properties is a bad practice, maybe there is a way to change it in the specific context that you need it.